src/client/pages/Geo/GeoMapWrapper/geoMapWrapper.scss

```diff
@@ -1,33 +1 @@
 @import 'src/client/style/partials';
```

Comment:
This file wasn't deleted because I will use it in an upcoming PR to add styles to the error message when the map fails to load. I tried deleting it and renaming it with the capital letter -> GeoMapWrapper.scss, but this would cause the same issue we had before with "Fileusages.scss".